### PR TITLE
Installation usability issues

### PR DESCRIPTION
**Pull request by ccarterlandis**
## Changelog

This PR:

- updates the installation process to write the all logs (including errors) to both `stdout` and to the `logs/` directory
- hides user input when entering their database password during the installation process
- adds a warning in the documentation to not use `sudo` when running or installing Augur

### CODE REVIEW

**Comment by sgoggins** [comment]
@ccarterlandis : Let me know if this is ready for review.

**Comment by ccarterlandis** [comment]
@sgoggons These all address @Nebrethar's concern, so I would like him to be the primary reviewer

**Comment by sgoggins** [comment]
@ccarterlandis : Rodger that.

File: scripts/install/config.sh [hunk 1]
```diff
@@ -72,7 +72,8 @@ function set_db_credentials() {
 
   read -p "Database: " db_name
   read -p "User: " db_user
-  read -p "Password: " password
+  read -s -p "Password: " password
```
Comment (Nebrethar) [inline scripts/install/config.sh:75]: Thank goodness for built-in flags!

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -56,14 +56,17 @@ after which you'll move on to the next section to configure the workers.
 .. note::
   Lines that start with a ``$`` denote a command to be run in an interactive terminal.
 
+.. warning::
+  Do **NOT** install or run Augur using ``sudo``. It is not required, and using it will inevitably cause some permissions trouble. Don't say we didn't warn you!
```
Comment (Nebrethar) [inline docs/source/getting-started/installation.rst:60]: Great 👍

File: docs/source/getting-started/installation.rst [hunk 3]
```diff
@@ -92,7 +95,7 @@ your installation of Python 3: on most systems, this is ``python3``, but yours m
 
    $ make install
 
-If you think something went wrong, check the log files under ``logs/install/``. If you want to try again, you can use ``make clean`` to delete any build files before running ``make install`` again.
+If you think something went wrong, check the log files in ``logs/``. If you want to try again, you can use ``make clean`` to delete any build files before running ``make install`` again.
```
Comment (Nebrethar) [inline docs/source/getting-started/installation.rst:98]: The log restructure is a great solution for now. It keeps the "finding what went wrong" process simple

**Review by Nebrethar — APPROVED** [review]
Great work! This makes the installation process more simple and clear

**Pull request merged**